src/tools/rbd/action/List.cc

```diff
@@ -101,6 +102,10 @@ int list_process_image(librados::Rados* rados, WorkerEntry* w, bool lflag, Forma
 
   std::vector<librbd::snap_info_t> snaplist;
   if (w->img.snap_list(snaplist) >= 0 && !snaplist.empty()) {
+    snaplist.erase(remove_if(snaplist.begin(),
```

Comment:
@dillaman  Hi, Jason. Shall we need to filter out the group/trash snapshots? or add a column like "NAMESPACE" (user/group/trash) to list all snapshots? Thanks.

Reply:
@Songweibin I'm fine w/ just filtering them out, thx.